tutorials/index.md

```diff
@@ -311,7 +311,7 @@ Increase the swap space to approximately 4-5GB. If your available space is less
     - Check `configuration='cluster'`
   - Launch Jupyter on cluster (steps and screen shots [here](https://github.com/brainiak/tutorials/wiki/Launching-notebooks-from-a-cluster) for Jupyter on server)
 
-    `cd ~/tutorials/tutorials`
+    `cd ~/brainiak-tutorials/tutorials`
```

Comment:
Could just assume current directory rather than home directory right? We don't know where they will run git clone from but presumably we can assume they will stay in that folder right?